modtools/item-trigger.lua

```diff
@@ -24,10 +24,26 @@ Arguments::
             RING
     -onStrike
         trigger the command on appropriate weapon strikes
-    -onEquip
+    -onEquip mode
         trigger the command when someone equips an appropriate item
-    -onUnequip
+        Optionally, the equipment mode can be specified
+        Possible values for mode:
+            0 = Hauled
```

Comment:
Is this a DF enum? If so, you should use human-readable names instead of making people look up numbers every time they want to want to read/write a command.

Reply:
On second glance, it looks like you just documented this. Not a big deal, then, but it would be nice to be able to use enum values (if this is in fact an enum).

Reply:
Noted. I suppose it'd cause far less hassle to everyone involved.